### ISSUE TITLE
error message not present when there is syntax error

### ISSUE THREAD

**Issue by ruanyl** [jscs-dev/node-jscs#2140]
when there is a syntax error, not style (error/warning). The error `message` is not present at reporter output. for example:

```
jscs --reporter json test.js --no-colors
{"test.js":[{"line":11,"column":20}]}
```

which actually there is a syntax error:

```
SyntaxError: Unexpected token ILLEGAL
```

Is it an expect behavior or a bug?

**Comment by xavierchow** [comment]
I have the same problem, I'm using [syntastic](https://github.com/scrooloose/syntastic) with jscs and it expects `message` from jscs...

**Comment by ruanyl** [comment]
It was actually fixed in https://github.com/jscs-dev/node-jscs/commit/b47f765546cfca93c0aa93a5e9c12d1d731d6e75

But it's only in `2.x` release, so install via `npm install jscs` won't get that fix.
@xavierchow

**Comment by xavierchow** [comment]
@ruanyl thanks for reminding, I think the fix is in 3.0 branch, no sure when there'll be a release for it tho.
